### PR TITLE
add "UDEV-gothic" font

### PR DESCRIPTION
**Pull request by justforlxz**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-fonts/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by justforlxz** [comment]
Hi, there are two questions I can't process.

1. brew style --cask

```shell
$ brew style --fix font-udev-gothic
Inspecting 2 files
.C

Offenses:

/opt/homebrew/Caskroom/font-udev-gothic/.metadata/0.0.7/20220329064348.040/Casks/font-udev-gothic.rb:1:1: C: [Corrected] Missing frozen string literal comment.
cask "font-udev-gothic" do
^
/opt/homebrew/Caskroom/font-udev-gothic/.metadata/0.0.7/20220329064348.040/Casks/font-udev-gothic.rb:2:1: C: [Corrected] Add an empty line after magic comments.
cask "font-udev-gothic" do
^

2 files inspected, 2 offenses detected, 2 offenses corrected
```

2. brew audit --new-cask

```shell
$ brew audit --new-cask font-udev-gothic
Error: Failed to load formula: font-udev-gothic
font-udev-gothic: undefined method `cask' for Formulary::FormulaNamespace71085f2efdf388a5a3643c7406e36f61:Module
Did you mean?  case
Warning: Treating font-udev-gothic as a cask.
==> Downloading https://github.com/yuru7/udev-gothic/releases/download/v0.0.7/UDEVGothic_v0.0.7.zip
Already downloaded: /Users/lxz/Library/Caches/Homebrew/downloads/8a050c45478005137b9afdc519c325cdefa5345a80678a194c1d9c460b717a82--UDEVGothic_v0.0.7.zip
audit for font-udev-gothic: failed
 - GitHub repository too new (<30 days old)
Error: 1 problem in 1 cask detected
```

File: Casks/font-udev-gothic.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+cask "font-udev-gothic" do
+  version "0.0.7"
```
Comment (bevanjkay) [inline Casks/font-udev-gothic.rb:2]: @justforlxz Can you please bump this to the latest version`0.0.9` - the shasum will also need updating.

File: Casks/font-udev-gothic.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  url "https://github.com/yuru7/udev-gothic/releases/download/v#{version}/UDEVGothic_v#{version}.zip"
+  name "UDEV Gothic"
+  desc "Integrate fonts from BIZ UD Gothic and JetBrains Mono"
+  homepage "https://github.com/yuru7/udev-gothic/"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
```
Comment (bevanjkay) [inline Casks/font-udev-gothic.rb:13]: @justforlxz The livecheck block can be ommitted in this instance as `livecheck` will automatically infer the `:git` strategy from the url.

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/font-udev-gothic.rb (hunk 2 above)
Comment (justforlxz) [reply Casks/font-udev-gothic.rb]: hi, i have done the update, please check again, thanks!

**Review by bevanjkay — APPROVED** [review]

**Comment by justforlxz** [comment]
@bevanjkay I noticed that action has the same error as mine, how can I fix it?

```
GitHub repository too new (<30 days old)
```

**Pull request merged**